packages/components/src/components/es-loading-text/es-loading-text.css

```diff
@@ -6,4 +6,18 @@
     white-space: pre;
     color: transparent;
     user-select: none;
+    max-width: 100%;
+    animation: shimmer 1.5s infinite;
+}
+
+@keyframes shimmer {
+    0% {
+        opacity: 0.5;
+    }
+    50% {
+        opacity: 1;
+    }
+    100% {
+        opacity: 0.5;
+    }
```

Comment:
I think if we are to add this, it should be more subtle. I'm not too keen on multiple things animating, especially in unison. I think for now, lets just leave it static (aside from the initial fade in) and possibly revisit it in the future.